### PR TITLE
Added reset functionality to error_handling.py

### PR DESCRIPTION
**Pull request by tavisit**

### CODE REVIEW

File: qf_lib/common/utils/error_handling.py [hunk 1]
```diff
@@ -55,4 +55,20 @@ def wrap(wrapped_cls):
...
+        """
+        Method used to get the error messages
+        """
         return cls._error_messages
+
+    @classmethod
+    def reset_error_messages(cls, show_old_messages: bool = False):
+        """
+        Method used to reset the error messages to their initial value
+        @param show_old_messages:bool
```
Comment (myrmarachne) [inline qf_lib/common/utils/error_handling.py:67]: Please use the same doc strings notation as in other files to make it available in sphinx

Comment (tavisit) [reply qf_lib/common/utils/error_handling.py]: Done

File: qf_lib/common/utils/error_handling.py [hunk 2]
```diff
@@ -55,4 +55,22 @@ def wrap(wrapped_cls):
 
     @classmethod
     def get_error_messages(cls):
+        """
```
Comment (myrmarachne) [inline qf_lib/common/utils/error_handling.py:58]: Method used to get the error messages

        Returns
        -------
        <here type - probably typing.List in this case>
                    current messages that are stored in the class

**Review by myrmarachne — APPROVED** [review]

File: qf_lib/common/utils/error_handling.py (hunk 2 above)
Comment (tavisit) [reply qf_lib/common/utils/error_handling.py:58]: Done

**Pull request merged**